### PR TITLE
[SEM-063]: AUTOR ART.

### PR DESCRIPTION
**Pull request by davidcm1011**
Como usuario quiero que se muestre informacion acerca del autor del art.

### CODE REVIEW

**Comment by DanX3N0** [comment]
FIGMA:
image

PR:
image

**Comment by DanX3N0** [comment]
the strokes of the letters should be thinner.

**Comment by DanX3N0** [comment]
FIGMA:
image

PR:
image

**Comment by DanX3N0** [comment]
image

image

check the props of the typography in figma

**Comment by DanX3N0** [comment]
PR:
image
image
figma:
image

it's according to figma

File: src/ui/components/atoms/Author-Label/styles.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+@import '../../../../globals/breakpoints';
+@import '../../../../globals/mixins';
+@import '../../../../globals/variables';
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@200;275;300&display=swap');
```
Comment (Ditmar) [inline src/ui/components/atoms/Author-Label/styles.scss:4]: We already have these fonts embedded in the application use the embed fonts directly don't use google fonts

**Review by Ditmar — CHANGES_REQUESTED** [review]

File: src/ui/components/atoms/label-references/LabelReferences.tsx [hunk 2]
```diff
@@ -3,17 +3,17 @@ import type { IProps } from './types/IProps';
...
 
   const getBackgroundClassName = () => {
-    return colorVariant === 'main' ? 'label-references--main-bg' : '';
+    return colorVariant === 'main' ? 'label-references-main-bg' : '';
   };
 
   return (
-    <div className={`label-references ${getColorClassName()} ${getBackgroundClassName()}`}>
-      <span className="label-references__text">{text}</span>
-    </div>
```
Comment (Ditmar) [inline src/ui/components/atoms/label-references/LabelReferences.tsx:16]: why are you changing the styles of this component?

**Review by Ditmar — CHANGES_REQUESTED** [review]

File: src/ui/components/atoms/label-references/LabelReferences.tsx (hunk 2 above)
Comment (davidcm1011) [reply src/ui/components/atoms/label-references/LabelReferences.tsx]: I don't remember modifying that component, but I have already corrected it

**Review by Ditmar — APPROVED** [review]

**Pull request merged**